src/main/resources/userEmailTemplate-feedbackSessionClosed.html

```diff
@@ -0,0 +1,13 @@
+Hello ${userName},
+
+The following feedback session ${status}.
```

Comment:
You don't need the placeholder `${status}` if there is only one usage of this template.